pkgs/games/gargoyle/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, fetchFromGitHub, jam, pkgconfig, gtk2, SDL, SDL_mixer, SDL_sound, smpeg, libvorbis }:
+
+stdenv.mkDerivation {
+  name = "gargoyle-2017-03-12";
+
+  src = fetchFromGitHub {
+    owner = "garglk";
+    repo = "garglk";
+    rev = "7d8fe875927c332ea30d75656bc982354b42f602";
+    sha256 = "00nac7j84gfql1cr11cs9ad4j1gsxrnkqfx6mis4ijf7d6px3sih";
+  };
+
+  nativeBuildInputs = [ jam pkgconfig ];
+
+  buildInputs = [ gtk2 SDL SDL_mixer ] ++ (
+    if stdenv.isDarwin then [ smpeg libvorbis ] else [ SDL_sound ]
+  );
+
+  patches = [ ./darwin.patch ];
+
+  preBuild = if stdenv.isDarwin then ''
+    export NIX_CFLAGS_COMPILE="$NIX_CFLAGS_COMPILE -I${SDL.dev}/include/SDL"
+    export GARGLKINI="$out/Applications/Gargoyle.app/Contents/Resources/garglk.ini"
+  '' else ''
+    export NIX_LDFLAGS="$NIX_LDFLAGS -rpath $out/libexec/gargoyle"
+    export DESTDIR="$out"
+    export _BINDIR=libexec/gargoyle
+    export _APPDIR=libexec/gargoyle
+    export _LIBDIR=libexec/gargoyle
+    export GARGLKINI="$out/etc/garglk.ini"
+  '';
+
+  buildPhase = "jam -j$NIX_BUILD_CORES";
+
+  installPhase = if stdenv.isDarwin then (builtins.readFile ./darwin.sh) else ''
+    jam -j$NIX_BUILD_CORES install
+    mkdir -p "$out/bin"
+    ln -s ../libexec/gargoyle/gargoyle "$out/bin"
```

Comment:
Are you sure that you built the current revision? Last Travis check confirms that libexec is there: https://travis-ci.org/NixOS/nixpkgs/jobs/212810559#L3500